independent-publisher-connectors/AfterShip/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2690 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "AfterShip",
+    "description": "AfterShip - Shipment tracking API"
```

Comment:
@taiki-yoshida Please provide a valid description for your connector which can be easily understandable by everyone and it should describe concisely the main purpose and value offered by your connector.

Docs - [Write a description for your connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)

Reply:
@taiki-yoshida The description of your connector still shows the same. Please do not resolve the comment until you push the changes to your PR.